src/plane.rs

```diff
@@ -100,86 +98,26 @@ pub struct PlaneOffset {
 ///
 /// The buffer is padded and aligned according to the architecture-specific
 /// SIMD constraints.
-#[derive(Debug, PartialEq, Eq)]
-#[cfg_attr(not(feature = "serialize"), derive(Serialize, Deserialize))]
+#[derive(Debug, Clone, PartialEq, Eq)]
+#[cfg_attr(feature = "serialize", derive(Serialize, Deserialize))]
 pub struct PlaneData<T: Pixel> {
-    ptr: std::ptr::NonNull<T>,
-    _marker: PhantomData<T>,
-    len: usize,
+    data: AVec<T, RuntimeAlign>,
```

Comment:
We can use [ConstAlign](https://docs.rs/aligned-vec/0.5.0/aligned_vec/struct.ConstAlign.html#) since we align to DATA_ALIGNMENT.